packages/pytest-simcore/src/pytest_simcore/pydantic_models.py

```diff
@@ -134,7 +170,12 @@ def model_cls_examples(model_cls: type[BaseModel]) -> dict[str, dict[str, Any]]:
     """
     Extracts examples from pydantic model class Config
     """
-
+    warnings.warn(
+        "The 'model_cls_examples' fixture is deprecated and will be removed in a future version. "
+        "Please use 'iter_model_example_in_class' or 'iter_model_examples_in_module' as an alternative.",
+        DeprecationWarning,
+        stacklevel=2,
+    )
```

Comment:
If you feel like it has to go away, I would just remove this one. Would not bother with deprecation warnings.

Reply:
@GitHK i think deprecation warnings are very important to spot what has to be updated. I do not understand why you think otherwise, can you elaborate?